src/chmframe.cpp

```diff
@@ -839,44 +855,49 @@ void CHMFrame::SaveExitInfo()
     _fh.Save(config);
 }
 
+#ifndef __WXGTK__
 namespace {
-
-#include <back.xpm>
-#include <copy.xpm>
-#include <fileopen.xpm>
-#include <find.xpm>
-#include <forward.xpm>
 #include <fullscreen.xpm>
-#include <helpicon.xpm>
-#include <home.xpm>
 #include <htmoptns.xpm>
 #include <htmsidep.xpm>
-#include <print.xpm>
-
```

Comment:
Please see previous comment on removing these newlines.

Reply:
Fixed with the latest commit.